src/services/activityReports.js

```diff
@@ -228,7 +228,92 @@ export function activityReports() {
       },
       {
         model: User,
-        attributes: ['name', 'role', 'fullName'],
+        attributes: ['id', 'name', 'role', 'fullName'],
+        as: 'collaborators',
+        through: { attributes: [] },
+      },
+    ],
+  });
+}
+/**
+ * Retrieves alerts based on the following logic:
+ * One or both of these high level conditions are true -
+ * manager - approvingManagerId matches and report status is submitted.
+ * specialist - author id or one of the collaborator's id matches and status is not approved nor
+ * submitted.
+ * @param {*} userId
+ */
+export function activityReportAlerts(userId) {
+  return ActivityReport.findAll({
+    where: {
+      [Op.or]: [
+        {
+          [Op.or]: [{ status: 'submitted' }, { status: 'needs_action' }],
```

Comment:
I think this should just be submitted, so that when I submit my review the report gets removed from my alerts. Thoughts?

Reply:
After talking about this with @kryswisnaskas, we're going to leave this for now. If this muddies the list we can create a new story to address.